<ui version="4.0" >
 <class>pickpointsDialog</class>
 <widget class="QWidget" name="pickpointsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>370</y>
      <width>411</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="removeHighlightedButton" >
       <property name="text" >
        <string>Remove Highlighted</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton" >
       <property name="text" >
        <string>Clear All Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renamePointButton" >
       <property name="text" >
        <string>Rename Point</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTreeWidget" name="pickedPointsTreeWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <property name="columnCount" >
     <number>4</number>
    </property>
    <column>
     <property name="text" >
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>4</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title" >
     <string>Mode: </string>
    </property>
    <widget class="QRadioButton" name="pickPointModeRadioButton" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text" >
      <string>Pick Point</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="movePointRadioButton" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text" >
      <string>Move Point</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QPushButton" name="saveButton" >
       <property name="text" >
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QPushButton" name="loadPointsButton" >
       <property name="text" >
        <string>Load Points From File</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>430</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title" >
     <string>Template Controls</string>
    </property>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>404</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="saveTemplateButton" >
        <property name="text" >
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadTemplateButton" >
        <property name="text" >
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearTemplateButton" >
        <property name="text" >
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPointToTemplateButton" >
        <property name="text" >
         <string>Add Point to Template</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCheckBox" name="defaultTemplateCheckBox" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>410</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Save this as your default template</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
